share/instruments/instruments.xml

```diff
@@ -6923,7 +6923,7 @@
                         <head>normal</head>
                         <line>0</line>
                         <voice>0</voice>
-                        <name>Low Agogo</name>
+                        <name>Brake Drum</name>
```

Comment:
Not "Automobile Brake Drums"? Should result in one string less to translate

Reply:
If not, we'd need to enhance the generateTS.py ro also extract those strings from instruments.xml, else it's end up untranslated

Reply:
We don't want drum names to get too long if we can help it because we're planning to display them in the drum palette below each note icon. We will need to enhance `generateTS.py` to extract drum names because they don't always match the drumset instrument name (many drumsets have multiple drums).

`generateTS.py` needs a lot of work before release (e.g. to account for how instrument names are constructed from multiple components in MU4, and to allow the translation of an identical string to be different for different instruments - i.e. the instrument ID must be given as context to the `tr()` function).

Reply:
OK, good to know that this is in scope